src/pytest_mock_resources/action.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
```

Comment:
I decided to split out the Actions as a concept from being sqlalchemy-specific, because i'm clearly reusing them here.

It's not strictly necessary, given that a given set of actions class objects are almost certainly only intercompatible with the actions on the fixture they're being used for.

But it seemed to make sense to me that the API/systems using them would be common amongst the fixture types that support them.